test/integration/script-loader/test/index.test.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-env jest */
+
+import { join } from 'path'
+import {
+  renderViaHTTP,
+  nextServer,
+  startApp,
+  stopApp,
+  nextBuild,
+  waitFor,
+} from 'next-test-utils'
+import webdriver from 'next-webdriver'
+import cheerio from 'cheerio'
+
+jest.setTimeout(1000 * 60 * 5)
+
+let appDir = join(__dirname, '..')
+let server
+let appPort
+
+describe('Script Loader', () => {
+  beforeAll(async () => {
+    await nextBuild(appDir)
+    const app = nextServer({
+      dir: appDir,
+      dev: false,
+      quiet: true,
+    })
+
+    server = await startApp(app)
+    appPort = server.address().port
+  })
+  afterAll(() => {
+    stopApp(server)
+  })
+
+  it('priority default', async () => {
+    let browser
+    try {
+      browser = await webdriver(appPort, '/')
+      const preloads = await browser.elementsByCss('link[rel=preload]')
+      const preloadEndHref = await preloads[preloads.length - 1].getAttribute(
```

Comment:
this is a bit fragile over the course of time.
can we expect this to be after `/next` preloads? instead of being the absolute last?